src/RecoveryServices/RecoveryServices.Backup.Autorest/docs/Edit-AzRecoveryServicesBackupSchedulePolicyClientObject.md

```diff
@@ -0,0 +1,454 @@
+---
+external help file:
+Module Name: Az.RecoveryServices
+online version: https://learn.microsoft.com/powershell/module/az.recoveryservices/edit-azrecoveryservicesbackupschedulepolicyclientobject
+schema: 2.0.0
+---
+
+# Edit-AzRecoveryServicesBackupSchedulePolicyClientObject
+
+## SYNOPSIS
+Edits the schedule policy in the specified backup policy object.
+
+## SYNTAX
+
+```
+Edit-AzRecoveryServicesBackupSchedulePolicyClientObject -Policy <IProtectionPolicy>
+ [-BackupFrequency <String>] [-DifferentialRunDay <String[]>] [-DifferentialScheduleTime <String>]
+ [-EnableDifferentialBackup <Boolean?>] [-EnableIncrementalBackup <Boolean?>] [-EnableLogBackup <Boolean?>]
```

Comment:
For the Boolean parameters, we recommend using Switch instead.

Reply:
@vidai-msft These parameter is used for enabling as well as disabling the corresponding properties depending on their boolean values so I think boolean is required in this case. Switch will not work directly as when the user doesn't pass the value, we probably don't want to disable it automatically.